frontend/src/components/GoalForm/ObjectiveStatus.js

```diff
@@ -17,10 +16,9 @@ export default function ObjectiveStatus({
     return null;
   }
 
-  // if the objective has been completed or is "in progress"
-  // we need a control to change status
-
-  const showDropdown = !(status.toLowerCase() === 'not started' && isOnReport);
+  // if the objective is "in progress" we need a control to change status
+  // or if the goal is 'closed'
+  const showDropdown = status === 'In Progress' && goalStatus !== 'Closed';
```

Comment:
Should this be goalStatus === 'Closed'? Im just wondering based on the above comment.